consent-ws/src/main/java/org/broadinstitute/consent/http/resources/ConsentsResource.java

```diff
@@ -85,4 +99,28 @@ protected Logger logger() {
         return Logger.getLogger("ConsentsResource");
     }
 
+
+    @POST
+    @Path("group-names")
+    @Produces(value = {MediaType.APPLICATION_JSON, MediaType.MULTIPART_FORM_DATA})
+    @Consumes(value = {MediaType.MULTIPART_FORM_DATA, MediaType.APPLICATION_JSON})
+    @RolesAllowed("ADMIN")
+    public Response updateGroupName(@Context UriInfo info, @Auth User user,
+                                    @FormDataParam("data") InputStream uploadedDataSet,
```

Comment:
not sure what the annotations actually do, but is it correct for both this and the one below to be data?